development-defaults.ini

```diff
@@ -68,21 +72,53 @@ attendee_badge = 3000, 29999
 one_day_badge = 30000, 39999
 
 
```

Comment:
One thing I did was move the ``age_groups`` stuff out of the database and into flatfile config, which ducks several of the annoying issues we ran into the other way.  (This would move out of flatfile and into Redis if/when that happens.)

Overall this is being done in mostly the same way with all of the same things being checked.